site/content/docs/manifest/lb-web-service.en.md

```diff
@@ -42,13 +42,16 @@ List of all available properties for a `'Load Balanced Web Service'` manifest. T
           requests: 10000
           response_time: 2s
         exec: true
-    
+
+        deployment:
```

Comment:
nit: can we remove the `recreate` deployment from the top-level fields in the LBWS, and Worker service manifests? Users might think we're encouraging the use of that field by default